### PR TITLE
Add atom feeds to travel advice

### PR DESCRIPTION
**Pull request by fofr**
https://trello.com/c/mNRwOtHL/139-2-render-travel-advice-atom-feeds-in-government-frontend-m

Render atom feeds for travel advice countries. Output of atom feed is identical to multipage-frontend’s (and live). Template and tests ported from multipage-frontend.

__When reviewing please double-check the routing logic and tests__

Add support for rendering atom routes:
* Update routing to allow atom format
* Use a regex to constrain matches on locale and format
* Test that all supported locales work with regex
* Append the locale when matched to the content_item_path
* Return 406 when unacceptable format requested

Paired with @andrewgarner

Example feed:
Matches: https://www.gov.uk/foreign-travel-advice/thailand.atom

```
<?xml version="1.0" encoding="UTF-8"?>
<feed xml:lang="en-US" xmlns="http://www.w3.org/2005/Atom">
  <id>https://www.gov.uk/foreign-travel-advice/thailand</id>
  <link rel="alternate" type="text/html" href="https://www.gov.uk/foreign-travel-advice/thailand"/>
  <link rel="self" type="application/atom+xml" href="http://government-frontend.dev.gov.uk/foreign-travel-advice/thailand.atom"/>
  <title>Travel Advice Summary</title>
  <updated>2017-02-08T10:23:34+00:00</updated>
  <author>
    <name>GOV.UK</name>
  </author>
  <entry>
    <id>https://www.gov.uk/foreign-travel-advice/thailand#2017-02-08T10:23:34+00:00</id>
    <updated>2017-02-08T10:23:34+00:00</updated>
    <link rel="alternate" type="text/html" href="https://www.gov.uk/foreign-travel-advice/thailand"/>
    <title>Thailand travel advice</title>
    <link rel="self" type="application/atom+xml" href="https://www.gov.uk/foreign-travel-advice/thailand.atom"/>
    <summary type="xhtml">
      <div xmlns="http://www.w3.org/1999/xhtml">
<p>Latest update: Summary – updated guidance for visitors following the passing of His Majesty King Bhumibol Adulyadej
</p>      </div>
    </summary>
  </entry>
</feed>
```

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
-  end
+  get 'healthcheck', to: proc { [200, {}, ['']] }
+
+  # FIXME: Update when https://trello.com/c/w8HN3M4A/ is ready
+  get 'foreign-travel-advice/:country/:part' => 'travel_advice#show'
+
+  get '*path(.:locale)(.:format)' => 'content_items#show',
+      constraints: {
+        format: /atom/,
+        locale: /\w{2}(-[\d\w]{2,3})?/,
```
Comment (nickcolley) [inline config/routes.rb]: This regex is gnarly for my simple monkey brain, should we have a few tests around this?

Comment (fofr) [inline config/routes.rb]: There is a test: https://github.com/alphagov/government-frontend/pull/257/files#diff-7dd9638113bd162736166ccac21fae7cR25

It tests every supported locale against the regex. If we add a new locale that's not supported, the test will fail.

File: config/routes.rb [hunk 2]
```diff
@@ -2,10 +2,14 @@
...
-    # FIXME: Update when https://trello.com/c/w8HN3M4A/ is ready
-    r.get 'foreign-travel-advice/:country/:part' => 'travel_advice#show'
-    r.get '*path' => 'content_items#show', constraints: { path: %r[.*] }
-  end
+  get 'healthcheck', to: proc { [200, {}, ['']] }
+
+  # FIXME: Update when https://trello.com/c/w8HN3M4A/ is ready
+  get 'foreign-travel-advice/:country/:part' => 'travel_advice#show'
+
+  get '*path(.:locale)(.:format)' => 'content_items#show',
```
Comment (nickcolley) [inline config/routes.rb]: It looks like the only thing that constrains this route is that it's got an /atom/ and a valid locale.

What happens when you try to render something that is not a travel advice format? Does this work?

Comment (fofr) [inline config/routes.rb]: It returns a 406: ba4a046 (ie not an accepted format)
If the templates were added, they would work.

File: app/presenters/travel_advice_presenter.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+require 'htmlentities'
```
Comment (fofr) [inline app/presenters/travel_advice_presenter.rb]: Hmm. I was following prior art: https://github.com/alphagov/multipage-frontend/blob/4bcfa99526be326888c04eaf12598785a6920e00/app/presenters/travel_advice_presenter.rb#L1

Comment (fofr) [inline app/presenters/travel_advice_presenter.rb]: Fixed and rebased.

**Pull request merged**